tfhe/src/core_crypto/gpu/vec.rs

```diff
@@ -10,9 +10,35 @@ use tfhe_cuda_backend::cuda_bind::{
     cuda_synchronize_device,
 };
 
```

Comment:
Minor: this empty line between imports could be removed
Applies to other places